.alexignore

```diff
@@ -45,9 +45,9 @@ website/versioned_docs/version-0.42/
 website/versioned_docs/version-0.43/
 website/versioned_docs/version-0.44/
 website/versioned_docs/version-0.46/
-website/versioned_docs/version-0.47/
-website/versioned_docs/version-0.51/
+website/versioned_docs/version-0.48/
+website/versioned_docs/version-0.49/
```

Comment:
48, 49, and 52 are also done - I think that when you're merging in master the conflicts are not correctly being fixed

Reply:
Did a pull and these were there :( uploading updated file